docs/source/api/docstrings/pydeseq2.dds.DeseqDataSet.rst

```diff
@@ -0,0 +1,21 @@
+﻿pydeseq2.dds.DeseqDataSet
```

Comment:
Hi @mandreux-owkin, I deactivated the autogeneration of rst files in [a3f9092](https://github.com/owkin/PyDESeq2/pull/66/commits/a3f9092f8809cdc2ef59af8a35aa9e939594bed0) to avoid the issue of documentation being created for attributes and methods inherited from anndata, which caused problems of readability and incompatible docstring formats.

Unless we find another solution and turn autogeneration on again, we should keep rst files in this PR.

Reply:
thanks for the explanation! So be it.